src/app/containers/Caption/__snapshots__/index.test.jsx.snap

```diff
@@ -85,10 +98,15 @@ exports[`should render caption text with example News offscreen text 1`] = `
   line-height: 1.125rem;
   font-family: ReithSans,Helvetica,Arial,sans-serif;
   font-weight: 400;
-  font-style: italic;
+  font-style: normal;
   color: #6E6E73;
-  padding: 0.5rem 0.5rem 0;
+  margin-top: 0.5rem;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
   width: 100%;
+  width: calc(100% - 0.5rem);
```

Comment:
Is this right having `width` declared twice?

Reply:
Also if you want any spacing around an element should we maybe use a `margin` of `0.5rem` rather than `padding`? In Simorgh the css [we normalise to use `border-box`](https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/global-styles.jsx#L7) as described [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) so the width will take into account the content, the padding but not the margin _I think_.

This approach would remove the need to calculate the width an subtract the padding? @simonsinclair @joshcoventry  tagging you as css experts here 🧙

Reply:
Thanks for the comments @andrewscfc,

- I defined `width` twice so that for browsers that don't support `calc` (albeit very few), there is a fallback, and with `width: 100%;` set the user wouldn't notice any visual difference, but there would be 0.5rem of hidden padding overflow.
- Initially I had no intention of using `padding`, and attempted to use `border-box` but couldn't think of a way to have variable spacing to the left and right of the grey line without the grey line becoming a semantic markup element instead of a CSS border.

Reply:
@andrewscfc, those are both valid points. Harry and I discussed the width 'fallback' early on and, in my opinion, it doesn't hurt to declare this - browsers like Opera Mini etc don't support calc. The padding is also necessary, because it is being used in combination with a margin to create space around a border.

Reply:
ah got ya, happy to approve as these approaches have clearly been thought about enough, thanks!